gce/burst-training/README.md

```diff
@@ -0,0 +1,242 @@
+# Compute Engine burst training
+
+[Google Compute Engine](https://cloud.google.com/compute/) gives you access to
+very powerful virtual machines which you can use for as much or, more
+importantly for most data science use cases, as little time as you need.
+
+Common tasks that might take tens of minutes or even hours on your personal can
+be completed in seconds on a machine with 64 cores and almost half a terabyte of
+memory, and this can be done at a cost of under $1.
+
+This makes Google Cloud Platform (or any equivalent cloud infrastructure for
```

Comment:
Unless we know that other cloud providers are truly equivalent (mostly from cost point of view), I would remove the sentence in parens. Also because:

1. We probably don't know if its really true
2. We want to use the benefit of doubt to our advantage. Why give folks any hint to try another cloud

Reply:
I know this is the case on AWS with spot EC2 instances (although I haven't compared costs).

I don't have strong feelings about mentioning AWS, etc. Just felt that it would be good to put over here for reasons of transparency.

Reply:
Removed reference to other providers.